partner_multi_company/migrations/15.0.1.1.0/post-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 ForgeFlow S.L. (http://www.forgeflow.com)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import SUPERUSER_ID, api
+
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    for user in env["res.users"].search([]):
```

Comment:
Don't duplicate code. Just import with `from ...hooks import post_init_hook` and call it here.

Reply:
Good remark

cc @ArnauCForgeFlow

Reply:
Done!